.eslintrc.json

```diff
@@ -1,20 +1,37 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 2017
-  },
   "extends": [
-    "eslint:recommended"
+    "eslint:recommended",
+    "plugin:security/recommended"
   ],
   "env": {
     "browser": true,
     "commonjs": true,
-    "es6": true
+    "es6": true,
+    "node": true
   },
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "plugins": [
+    "jsdoc",
+    "json",
+    "security"
+  ],
   "root": true,
   "rules": {
+    "jsdoc/check-types": "error",
+    "security/detect-non-literal-fs-filename": "off",
+    "security/detect-object-injection": "off",
+
+    "eqeqeq": "error",
     "indent": ["error", 2, {"SwitchCase": 1, "VariableDeclarator": {"var": 2, "let": 2, "const": 3}}],
     "linebreak-style": ["error", "unix"],
+    "no-var": "off",
```

Comment:
What do we need `var` for? I only see it in a couple places in this diff...

Reply:
`var ` is used in the build-utils files, which are also linted.

Reply:
Does it need to be `var` there, and if so, would it be better to just ignore the `no-var` rule on those lines?

Reply:
node.js still has some issues with `let` at the top-level.  I can go back through those files to fix it, but really rather file a chore for now.